test/e2e/cluster_test.go

```diff
@@ -107,7 +107,7 @@ var _ = Describe("Workload cluster creation", func() {
 		dumpSpecResourcesAndCleanup(ctx, cleanInput)
 	})
 
-	It("With 1 control-plane nodes and 1 worker nodes", func() {
+	It("Default CNI - with 1 control-plane nodes and 1 worker nodes", func() {
```

Comment:
This is to help allow use to run a single targeted test since "with 1 control-plane nodes and 1 worker nodes" is used in multiple tests